tools/utils.py

```diff
@@ -64,6 +64,9 @@ def GuessOS():
   elif id == 'NetBSD':
     return 'netbsd'
   elif id == 'AIX':
+    # platform.system() returns 'aix' on both IBMi and AIX
+    if platform.uname()[0] == 'OS400':
+      return 'ibmi'
```

Comment:
Hello @richardlau , I have updated the code, but the cases https://github.com/nodejs/node/pull/30819/files#diff-6bc2c4c4b9f44f85c00eada987beb2c9R37-R44 are not skipped. I was testing with `python2 tools/test.py -J`. Something wrong here?

Reply:
What does `platform.uname()` return in Python on IBMi?

Reply:
```
$ python2
Python 2.7.17 (default, Oct 31 2019, 11:14:37) 
[GCC 6.3.0] on aix7
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> platform.uname()
('OS400', 'UT28P60', '3', '7', '00100008584T', '')
>>> 
```

Reply:
Can you also check that `platform.system()` returns `AIX` on IBMi PASE and not `OS400`?

Reply:
It returns 'OS400' 🤣 
Thank you!